src/common/DeveloperConfig.ts

```diff
@@ -14,10 +14,13 @@ import { AppTile } from '../hooks/useAppConfig';
  * when an app tile with a matching URL is tapped.
  *
  * @param simpleTheme Allows for configuring a theme via a primary and accent color.
+ *
+ * @param baseURL Allows for configuring a custom base API URL
  */
 export type DeveloperConfig = {
   appTileScreens?: AppTileScreens;
   simpleTheme?: SimpleTheme;
+  baseURL?: string;
```

Comment:
Big picture, this looks great.

A nit-pick - in this very-high-level interface context, the name is a little vague.  Base URL for what, the web apps that launch from app tiles?  I could see this being `apiBaseURL` instead, even though it's indeed called `baseURL` for the two providers that take it.  

I could also see more hinting in the description about how this is probably not needed unless performing advanced debugging involving a dev mock server, or something to that effect.